proto/zitadel/object/v2beta/object.proto

```diff
@@ -96,6 +96,11 @@ message ListDetails {
       description: "the last time the projection got updated"
     }
   ];
+  uint64 position = 4 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "\"267831\"";
+    }
+  ];
```

Comment:
i think we should add a description, also to state what's the difference to sequence

Reply:
i don't really want to have to many other changes, but since we add it in the list, should we also add it in the details as well?

Reply:
Can do that